plugins/talk-plugin-local-auth/client/components/ChangeEmailContentDialog.js

```diff
@@ -16,24 +83,30 @@ class ChangeEmailContentDialog extends React.Component {
     });
   };
 
+  cancel = () => {
+    this.clearForm();
```

Comment:
Clear form does nothing

Reply:
Besides we don't need to clear the form, just close the dialog

Reply:
Clearing the state will not clear the form because, you are using uncontrolled input (not recommended)